data-raw/process_data_sets_maggie.R

```diff
@@ -0,0 +1,266 @@
+
+tv_hurricanes <- read_csv("data-raw/puerto-rico-media/tv_hurricanes.csv")
+tv_hurricanes <- clean_names(tv_hurricanes)
+tv_hurricanes$date <- as.Date(tv_hurricanes$date, format= "%m / %d / %y")
+devtools::use_data(tv_hurricanes, overwrite = TRUE)
+
+tv_hurricanes_by_network <- read_csv("data-raw/puerto-rico-media/tv_hurricanes_by_network.csv")
+tv_hurricanes_by_network <- clean_names(tv_hurricanes_by_network)
+tv_hurricanes_by_network <- tv_hurricanes_by_network %>%
+  mutate(
+    date = as.Date(date, format= "%m / %d / %y"),
+    query = as.factor(query)
+  )
+devtools::use_data(tv_hurricanes_by_network, overwrite = TRUE)
+
+tv_states <- read_csv("data-raw/puerto-rico-media/tv_states.csv")
+tv_states <- clean_names(tv_states)
+tv_states$date <- as.Date(tv_states$date, format= "%m / %d / %y")
+devtools::use_data(tv_states, overwrite = TRUE)
+
+# riddler-pick-lowest---------------------------------------------------------------------
+riddler_pick_lowest <- read_csv("data-raw/riddler-pick-lowest/low_numbers.csv")
+riddler_pick_lowest <- clean_names(riddler_pick_lowest)
+devtools::use_data(riddler_pick_lowest, overwrite = TRUE)
+
+# sandy-311-calls---------------------------------------------------------------------
+sandy_311 <- read_csv("data-raw/sandy-311-calls/sandy-311-calls-by-day.csv")
+sandy_311 <- clean_names(sandy_311)
+
+sandy_311 <- sandy_311 %>%
+  rename(
+    nyc_311 = nyc_3_1_1,
+    nyc_service = nycservice,
+    nys_emergency_mg = nysemergencymg
+  ) %>%
+  mutate(
+    date = as.Date(date, format= "%m / %d / %y")
+  )
+sandy_311<-gather(sandy_311, "agency", "num_calls", 2:24)
+sandy_311$agency <- as.factor(sandy_311$agency)
+devtools::use_data(sandy_311, overwrite = TRUE)
+
+# trump-approval-ratings---------------------------------------------------------------------
+trump_approval_poll <- read_csv("data-raw/trump-approval-ratings/approval_polllist.csv")
+trump_approval_poll <- trump_approval_poll %>%
+  mutate(
+    multiversions = ifelse(multiversions == "*", TRUE, FALSE),
+    multiversions = ifelse(is.na(multiversions), FALSE, TRUE),
+    tracking = ifelse(is.na(tracking), FALSE, TRUE),
+    subgroup = as.factor(subgroup),
+    modeldate = as.Date(modeldate, format = "%m / %d / %Y"),
+    startdate = as.Date(startdate, format = "%m / %d / %Y"),
+    enddate = as.Date(enddate, format = "%m / %d / %Y"),
+    pollster = as.factor(pollster),
+    grade = as.factor(grade),
+    population = as.factor(population),
+    url = as.factor(url),
+    createddate = as.Date(createddate, format = "%m / %d / %Y")
+  ) %>%
+  rename(
+    model_date = modeldate,
+    start_date = startdate,
+    end_date = enddate,
+    sample_size = samplesize,
+    created_date = createddate
+  )
+#TODO: unsure how to deal with timestamp-- split into date and time or leave as is?
+#TODO: can I delete "president" and "modeldate" since each only has one value in this dataset (trump and Jan 5)? if so:
```

Comment:
Yes, good job spotting this. A variable has to vary! Add as a note in the helpful that you deleted these however.